corehq/apps/locations/forms.py

```diff
@@ -532,9 +533,9 @@ def __init__(self, domain_object, location, *args, **kwargs):
             )
         )
 
-    @property
     @memoized
```

Comment:
The only thing with this is that until I get https://github.com/dimagi/commcare-hq/pull/19853 working (I need to look into test failures), `memoized`'s local cache is scoped to the current request and `quickcache`'s isn't. Since the point of this change was more to cache it for the following request, I think we decided just not to mess with that if I recall. @pr33thi is that right / was there another reason?

Reply:
This has got me thinking that we could probably stand to have a docs page on the two.  @dannyroberts any interest in taking that on?  You'd certainly be the top pick for that, but I'd be happy to take a crack at it, or if you want to just dump some notes I could organize it.  I started a google doc with some specific points I'd like to see covered:
https://docs.google.com/document/d/1YzD4P1MLef6GMK13Uj51oMK4_HIK4jyRrydrfwYsnV0/edit#

Reply:
Good idea. I'll offline with you on where/how best to do that